cmd/util/ledger/migrations/storage_v6.go

```diff
@@ -461,18 +452,24 @@ func (m *StorageFormatV6Migration) initNewInterpreter() {
 		newInter.WithStorage(m.storage),
 		newInter.WithImportLocationHandler(
 			func(inter *newInter.Interpreter, location common.Location) newInter.Import {
-				program, err := m.loadProgram(location)
-				if err != nil {
-					panic(err)
+				var program *newInter.Program
+				if location == stdlib.CryptoChecker.Location {
+					program = newInter.ProgramFromChecker(stdlib.CryptoChecker)
```

Comment:
Good catch. I think next time we should try use the Cadence `runtime` package, which handles this already. We might have to extend it for such migration use-cases, it's currently mainly only providing functionality for the FVM/EN.